src/app/screens/BookDetail/components/CommentList/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, TouchableOpacity, Text } from 'react-native';
+import CommentCard from '@app/components/CommentCard';
+import { Comment } from '@interfaces/comment';
+import { COMMENTS } from '@constants/mockData';
+
+import styles from './styles';
+
+function CommentList() {
+  const renderComment = (comment: Comment) => <CommentCard key={comment.id} {...comment} />;
+  return (
+    <View style={styles.commentSection}>
+      <View>{COMMENTS.map(renderComment)}</View>
+      <TouchableOpacity style={styles.viewAllButton}>
+        <Text style={styles.viewAllButtonText}>View All</Text>
+      </TouchableOpacity>
```

Comment:
What happens when the user taps this button?

Reply:
Now, I undestand the point